image_tools/README.md

```diff
@@ -0,0 +1,33 @@
+## **What Is This?**
+
+This demo provides a simple utility to **connect to a workstation default camera device** and **display** it in an **OpenCV** window like below:
+
+![](img/result.png)
+
+## **Build**
+
+```bash
+colcon build --package-select image_tools
```

Comment:
```suggestion
colcon build --packages-select image_tools
```

Reply:
Done.

Reply:
Did you forget to push or commit something?  Because I don't see this change (or any of the other ones) reflected in the diff.

Reply:
Apologies. Realized that I made commits to the wrong branch. Rectifying shortly.